docusaurus.config.ts

```diff
@@ -101,23 +101,39 @@ const config: Config = {
           customCss: [require.resolve("./src/css/custom.css")],
         },
         docs: {
-          sidebarPath: require.resolve("./sidebars.js"),
-          path: "opentofu-repo/website/docs",
+          includeCurrentVersion: false,
+          lastVersion: 'v1.6',
+          docVersionRootComponent: "@theme/DocVersionRoot",
+          versions: {
+            "v1.6": {
+              label: "1.6.x",
+              path: ""
+            },
+            "v1.7": {
+              label: "1.7.x",
+              path: "next",
+              banner: "unreleased",
+              noIndex: true
+            }/*,
+            "main": {
```

Comment:
In any case, I'd call this dev. I think it's more intuitive than main ("main" indicates this is the main docs to look at)

Reply:
I know this is commented out right now.